integration_tests/pages/prisonerList/PrisonerListPage.ts

```diff
@@ -76,6 +76,35 @@ export default class PrisonerListPage extends Page {
     return this
   }
 
+  hasPaginationControlsDisplayed(): PrisonerListPage {
+    this.paginationControls().should('exist')
+    return this
+  }
+
+  setPaginationNumber(value: number): PrisonerListPage {
+    this.paginationControls().find(`li:nth-of-type(${value}) a`).click()
+    return this
+  }
+
+  hasPreviousLinkDisplayed(): PrisonerListPage {
+    this.paginationControls().find('li:first-of-type a').should('contain', 'Previous')
+    return this
+  }
+
+  hasNextLinkDisplayed(): PrisonerListPage {
+    this.paginationControls().find('li:last-of-type a').should('contain', 'Next')
+    return this
+  }
+
+  hasNumberOfPagesDisplayed(expected: number): PrisonerListPage {
+    this.paginationControls()
+      .find('li.moj-pagination__item')
+      .not('.moj-pagination__item--next')
```

Comment:
Maybe not the best approach but only way I could think of without having the ability to pass attributes in the Nunjucks macros!

Reply:
It's fine; I agree, we don't want to litter the nunjucks markup with too many attributes etc just to support tests.